progressbar.go

```diff
@@ -132,9 +132,9 @@ func OptionSetWidth(s int) Option {
 }
 
 // OptionSpinnerType sets the type of spinner used for indeterminate bars
-func OptionSpinnerType(spinnerType int) Option {
+func OptionSpinnerType(spinner []string) Option {
```

Comment:
This change forces a new version because its backward-incompatible. Could you instead update `OptionSpinnerType(spinnerType int)` and make a new function `OptionSpinnerCustom(spinner []string)` ?

Reply:
Thanks for the feedback.
Makes total sense. Will do that in the coming days!

Reply:
Perfect!!! No rush. Very nice pr 😊

Reply:
I made the new option. Here are some additional things:

Because the 2 options cannot be used together (since one will always override the other), I made a small change in the `Add()` func: it returns an `error` if both options are used. 

This is not ideal and the correct place would be to make the `Option` return an `error` and check this error when creating a new Bar. But since this is also a breaking change (then we have 2 return params and we would have to at least omit the error using "_" in existing code) I did not go down that road - that is something for the next version.

Nonetheless, I felt _some_ error checking would be good 😄

Reply:
This looks fantastic, thanks for making the change! Such a great addition :)